### ISSUE TITLE
Rails 5.2. ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin18]

### ISSUE THREAD

**Issue by Evshved** [guard/guard#920]
```
bundle exec guard --debug
23:17:38 - DEBUG - Notiffany is using TerminalNotifier to send notifications.
23:17:39 - DEBUG - Command execution: hash stty
23:17:39 - DEBUG - Guard starts all plugins
23:17:39 - DEBUG - Hook :start_begin executed for Guard::RSpec
23:17:39 - INFO - Guard::RSpec is running
23:17:39 - DEBUG - Hook :start_end executed for Guard::RSpec
23:17:39 - INFO - Guard is now watching at '/Users/shved/programming/Rubyroidlabs/auto-city'
23:17:39 - DEBUG - Start interactor
23:17:39 - DEBUG - Command execution: stty -g 2>/dev/null
#<Thread:0x00007fc26d0bddf8@/Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/guard-2.14.2/lib/guard/jobs/pry_wrapper.rb:102 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
	15: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/guard-2.14.2/lib/guard/jobs/pry_wrapper.rb:102:in `block (2 levels) in _switch_to_pry'
	14: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_class.rb:200:in `start'
	13: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:13:in `start'
	12: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:38:in `start'
	11: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/input_lock.rb:77:in `with_ownership'
	10: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/input_lock.rb:59:in `__with_ownership'
	 9: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:38:in `block in start'
	 8: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:67:in `repl'
	 7: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:67:in `loop'
	 6: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:68:in `block in repl'
	 5: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:95:in `read'
	 4: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_instance.rb:550:in `select_prompt'
	 3: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_class.rb:401:in `critical_section'
	 2: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_instance.rb:553:in `block in select_prompt'
	 1: from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_instance.rb:565:in `generate_prompt'
/Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/guard-2.14.2/lib/guard/jobs/pry_wrapper.rb:279:in `block in _prompt': undefined method `input_array' for #<Pry:0x00007fc26d0952b8> (NoMethodError)
Did you mean?  input_ring
23:17:39 - DEBUG - Interactor was stopped or killed
23:17:39 - DEBUG - Command execution: stty  2>/dev/null
/Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/guard-2.14.2/lib/guard/jobs/pry_wrapper.rb:279:in `block in _prompt': undefined method `input_array' for #<Pry:0x00007fc26d0952b8>
Did you mean?  input_ring (NoMethodError)
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_instance.rb:565:in `generate_prompt'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_instance.rb:553:in `block in select_prompt'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_class.rb:401:in `critical_section'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_instance.rb:550:in `select_prompt'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:95:in `read'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:68:in `block in repl'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:67:in `loop'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:67:in `repl'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:38:in `block in start'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/input_lock.rb:59:in `__with_ownership'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/input_lock.rb:77:in `with_ownership'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:38:in `start'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/repl.rb:13:in `start'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/pry-0.12.1/lib/pry/pry_class.rb:200:in `start'
	from /Users/shved/.rvm/gems/ruby-2.5.1@auto-city/gems/guard-2.14.2/lib/guard/jobs/pry_wrapper.rb:102:in `block (2 levels) in _switch_to_pry'
```

**Comment by Evshved** [comment]
```
    guard (2.14.2)
      formatador (>= 0.2.4)
      listen (>= 2.7, < 4.0)
      lumberjack (>= 1.0.12, < 2.0)
      nenv (~> 0.1)
      notiffany (~> 0.0)
      pry (>= 0.9.12)
      shellany (~> 0.0)
      thor (>= 0.18.1)
    guard-compat (1.2.1)
    guard-rspec (4.7.3)
      guard (~> 2.1)
      guard-compat (~> 1.1)
```

**Comment by Evshved** [comment]
OS version - macOS Mojave version 10.14.1

**Comment by rymai** [comment]
@Evshved This was fixed in Guard `2.15.0`: https://github.com/guard/guard/releases/tag/v2.15.0
